Core/Keyed/GameplayCommands.xml

```diff
@@ -643,9 +643,9 @@
   <!-- EN: Linked storage settings -->
   <LinkedStorageSettings>Propojená nastavení skladišť</LinkedStorageSettings>
   <!-- EN: 1 building -->
-  <OneBuilding>TODO</OneBuilding>
+  <OneBuilding>1 objekt</OneBuilding>
   <!-- EN: {0} buildings -->
-  <NumBuildings>ﾠ{replace: ^{0}^; "^1^"-"u 1 objektu"; "^{0}^"-"u {0} objektů"}</NumBuildings>
+  <NumBuildings>{replace: ^{0}^; "^1^"-"1 objekt"; "^2^"-"2 objekty"; "^3^"-"3 objekty"; "^4^"-"4 objekty"; "^{0}^"-"{0} objektů"}</NumBuildings>
```

Comment:
Nevím kde se používalo u X objektů, ale teď jsem narazil na `Skupina skladišť: Skupina 1 (u 10 objektů)` a tam mi to `u` nedávalo smysl

Reply:
popravdě ani já nevím, kde se vyskytovalo. asi to nech jak si opravil